lib/viewmodel/club_detail/club_detail_viewmodel.dart

```diff
@@ -43,6 +46,29 @@ class ClubDetailViewModel with ChangeNotifierEx {
 
   Future<void> getActivites(String clubId) async {
     activities = await _clubDetailRepository.getActivites(clubId);
```

Comment:
I can't seem to find a try catch anywhere?
What if something goes wrong in the repository?